src/devices/QwiicButton/QwiicButton.LedConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using Iot.Device.QwiicButton.RegisterMapping;
+
+namespace Iot.Device.QwiicButton
+{
+    public sealed partial class QwiicButton
+    {
+        /// <summary>
+        /// Turns the onboard LED on with the specified brightness.
+        /// <param name="brightness">LED brightness value between 0 (off) and 255 (max). Defaults to max.</param>
+        /// </summary>
+        public void LedOn(byte brightness = 255)
+        {
+            LedConfig(brightness, 0, 0);
+        }
+
+        /// <summary>
+        /// Turns the onboard LED off.
+        /// </summary>
+        public void LedOff()
+        {
+            LedConfig(0, 0, 0);
+        }
+
+        /// <summary>
+        /// Configures the onboard LED with the given max brightness, granularity, cycle time, and off time.
+        /// <param name="brightness">LED brightness value between 0 (off) and 255 (max). Default is 255.</param>
+        /// <param name="cycleTime">Total pulse cycle time (in ms). Does not include off time.</param>
```

Comment:
I assume for this and the other one/two arguments `0` has special meaning of `do not touch this value`, right? if so please make sure documentation mentions that. If no then please make sure SetBrightness either uses previous values or at minimum mentions that it's not gonna blink anymore

Reply:
Added more explanation.